doc/pages/Manual.md

```diff
@@ -178,6 +178,18 @@ Switches are laid out thusly:
 - `<switch-prefix>/.opam-switch/backup`: snapshots of previous states of the
   switch, and other backup files.
 
+#### Plugins
+
+When opam doesn't know a subcommand name (e.g. `opam plugin-name`), it looks at
+the binaries installed in `${OPAMROOT}/plugins/bin` (by default: `~/.opam/plugins/bin`).
+This directory contains symlinks to binaries installed in whichever opam switch
+the plugin it belongs to was installed in.
```

Comment:
```suggestion
This directory contains symlinks to binaries installed in the same directory where the plugin's opam switch
was installed.
```
Here is a suggestion that might be a little more clear (although I'm not sure if the plugin belongs to the opam switch or the opam switch belongs to the plugin). Clarify that, and we can reword as needed.